runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/BatchViewOverrides.java

```diff
@@ -1378,6 +1378,11 @@ public T getValue() {
       return value;
     }
 
+    @Override
+    public @Nullable Boolean isDraining() {
```

Comment:
Making this nullable might create annoyance downstream. Default to `false`? I know this means that we cannot represent older SDKs where we "don't know"